omnibus_build.sh

```diff
@@ -9,11 +9,22 @@
 PROJECT_DIR=dd-agent-omnibus
 PROJECT_NAME=datadog-agent
 LOG_LEVEL=${LOG_LEVEL:-"info"}
+JMX_VERSION=${JMX_BRANCH:-"current"}
+export AGENT_BRANCH=${AGENT_BRANCH:-"master"}
 export OMNIBUS_BRANCH=${OMNIBUS_BRANCH:-"master"}
 export OMNIBUS_SOFTWARE_BRANCH=${OMNIBUS_SOFTWARE_BRANCH:-"master"}
 export OMNIBUS_RUBY_BRANCH=${OMNIBUS_RUBY_BRANCH:-"datadog-5.5.0"}
 export INTEGRATION_CORE_BRANCH=${INTEGRATION_CORE_BRANCH:-"master"}
 
+REMOTE_AGENT_REPO_RAW="https://raw.githubusercontent.com/DataDog/dd-agent"
+LOCAL_DD_AGENT="/dd-agent-repo"
+
+unset_jmx() {
+    echo "JMX provided by dd-agent in selected branch, JMX environment variables will be ignored."
```

Comment:
Should we still support this case once the PR is merged? Shouldn't the jar always be pulled from the s3 bucket?
Removing support for this case would largely simplify the code below IMO, we could just fail the build if we can't find the version of jmxfetch in the dd-agent file.
What do you think?